.github/workflows/update-data.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       - name: Bundle changes
         if: steps.git-check.outputs.modified == 'true'
-        run: npm run grunt images
+        run: npm run grunt data
```

Comment:
Yeah, had a 🤦🏻‍♂️ moment as soon as I saw the logs for the recent update.  Thanks for cleaning this up.